util_test.go

```diff
@@ -67,4 +67,7 @@ func TestCheckSignature(t *testing.T) {
 	if err == nil {
 		t.Fatal("Expected an error verifying a request without a signature header")
 	}
+	if valid {
+		t.Fatal("Expected signature to be invalid")
```

Comment:
Just like the error messages - this should also be all lowercase with no sentence punctuation.